codecov.yml

```diff
@@ -1,2 +1,23 @@
+# Documentation: https://docs.codecov.io/docs/codecov-yaml
 ignore:
   - "api/v1alpha1/zz_generated.deepcopy.go" # generated file
+
+# general codecov settings
+codecov:
+    notify:
+        require_ci_to_pass: false
+coverage:
+    status:
+        # sets the expected status for `codecov/project` check.
+        # `target: auto` means it will use the current coverage to calculate against
+        #          the give PR/commit.
+        # theshould is the max delta allowed when compared with the current coverage
+        project:
+            default:
+                target:  auto
+                threshold: 3%
+        # sets the expected status for `codecov/patch` check
+        # we set this to be only informatinal hence it does not cause the check to fail.
+        patch:
+            default:
+                informational: true
```

Comment:
this sounds redundant with line 8

Reply:
I guess this was clarified in that comment. This makes specifically `codecov/patch` as informational only.